Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/ui/dashboard/MovieDashboardActivity.kt

```diff
@@ -23,7 +23,7 @@ class MovieDashboardActivity : AppCompatActivity(), MovieGridAdapter.MovieClickL
 
     private lateinit var moviesViewModel: MoviesViewModel
     private lateinit var moviesRecyclerView: RecyclerView
-    private val dataManager = MovieDataManager()
+    private val movieAdapter by lazy { MovieGridAdapter(emptyList(), this)  }
```

Comment:
You are still passing the list of movies through the constructor. I advised you not to do it in past PRs